src/FINDOLOGIC/Export/Data/Property.php

```diff
@@ -59,7 +61,16 @@ public function addValue($value, $usergroup = null)
             throw new DuplicateValueForUsergroupException($this->key, $usergroup);
         }
 
-        $this->values[$usergroup] = $value;
+        $this->values[$usergroup] = DataHelper::emptyValueCheck($value);
+    }
+
+    public function setValues($values)
```

Comment:
Is `public` visibility necessary?

Reply:
No. Bulk adding entries should be done when the property object is initialized. Additional entries can be added with the `addValue`-method later on if needed.